packages/zmarkdown/__tests__/__snapshots__/old-suite-latex.test.js.snap

```diff
@@ -21,11 +21,11 @@ This \\\\& that.
 
 
 
-Here's a link\\\\ref{1} with an ampersand in the URL.
+Here's a \\\\href{1}{link} with an ampersand in the URL.
```

Comment:
Two comments here:

1. I'm not sure this is the expected behaviour, you're basically putting the hyperlink on the word `link` rather of the `1` above that will be suppressed;
2. as far as I know, the `\href` command is not used to refer to the current document in LaTeX but rather intended for external links; maybe you're looking for `\hyperref`, but please check 1. first.

Reply:
`\ref` is not an hyperlink, or we need to redefine `\ref` on latex side.

Reply:
Not sure what you mean here, but I'm quite sure that:

- `\ref` is a link to the current document;
- `\href` is a link to an external ressource.

Reply:
- `\ref` is not a link, it's just a way to latex to print the reference in the format we asked (e.g figure 1 or Part 1) 
- `\href`is a link whatever the target, it's from hypertext package
- `externalLink` is a link to an external target, I don't know the package

As seen with @karnaj we will use `\hyperref[label]{printed text}`